operator/internal/controller/redpanda/redpanda_controller_test.go

```diff
@@ -455,6 +456,58 @@ func (s *RedpandaControllerSuite) TestClusterSettings() {
 	s.deleteAndWait(rp)
 }
 
+func (s *RedpandaControllerSuite) TestLicenseReal() {
+	license := os.Getenv("REDPANDA_SAMPLE_LICENSE")
```

Comment:
I think the envvar in CI is actually just `REDPANDA_LICENSE`. It's infuriating that the envvar of the secret is defined within the secret and it's actual name.

Reply:
Nope 
https://buildkite.com/redpanda/redpanda-operator/builds/6228#019681d8-6c87-46ae-843f-07ec80bf7eed/141-149